### ISSUE TITLE
Strip down dependencies to minimum

### ISSUE THREAD

**Issue by stoutheo** [ros-pybullet/ros_pybullet_interface#1]
- [ ] Move all examples .py and .launch that depend on other packages of [ros-pybullet](https://github.com/ros-pybullet) into a separate repo in the org. 
- [ ] Prepare an installation script for the new repo that gets all dependencies and packages for examples.
- [ ] Prepare an installation script that is only for [this](https://github.com/ros-pybullet/ros_pybullet_interface) repo (dependency on [custom_ros_tools](https://github.com/ros-pybullet/custom_ros_tools) will remain).

**Comment by joaomoura24** [comment]
I would dare here to add that over time you should try to remove the dependencies on EXOTica, which is what takes most of the installation time.

**Comment by cmower** [comment]
Yup, also planning that ik_ros will be replaced with [optas](https://github.com/cmower/optas).

**Comment by cmower** [comment]
Yup, also planning that ik_ros will be replaced with [optas](https://github.com/cmower/optas).
